src/membrane-keypad-element.ts

```diff
@@ -118,8 +148,10 @@ export class MembraneKeypadElement extends LitElement {
     `;
   }
 
-  private down(key: string) {
+  private down(key: string, event: UIEvent) {
     if (!this.pressedKeys.has(key)) {
+      const currTarget = event.currentTarget as SVGElement;
+      currTarget.classList.add('pressed');
```

Comment:
What's the reason you need the `pressed` class?
It seems like the `:active` selector that you have above already does the trick

Reply:
The active selector isn't triggered when pressing the space key, so I added this class. 
But it has an isssue - while pressing space, if you press tab and press space again or click with the mouse on another button, it will display multiple buttons as pressed. I can fix the tab option by removing the class when pressing tab. But I couldn't think of a way to fix the mouse selection. Do you think I should check every button on mousedown? Got a better idea?

Reply:
Good point! 

So perhaps you could:
1. For the tab issue - remove the `pressed` class on `blur` event. That's way, when you press the tab key, the element loses focus, and the highlight is gone.
2. Make the event parameter optional and only pass it on keyboard event, so it won't get added on mouse events (this will probably solve the issue with the class sticking around when you click on the button, and then drag the mouse pointer out of the element area, which the button is still pressed)